pyproject.toml

```diff
@@ -0,0 +1,40 @@
+[tool.poetry]
+name = "hysr"
+version = "0.0.1"
+description = "Learning table tennis from scratch"
+authors = [
+	"Vincent Berenz <vberenz@tuebingen.mpg.de>",
+        "Dieter Buechler <dieter.buechler@tuebingen.mpg.de>",
+	"Simon Guist <simon.guist@tuebingen.mpg.de>",
+	"Felix Widmaier <felix.widmaier@tuebingen.mpg.de>"
+]
+packages = [
+	 { include = "hysr" },
+	 { include = 'hysr_install' }
+] 
+include = ["hysr_install/pam.sh"]
+
+[tool.poetry.dependencies]
+python = ">=3.10,<3.11"
+colcon-common-extensions = "^0.2.1"
+treep = "^1.52"
+fyplot = "^2.0"
+PySide2 = "^5.15.2"
+empy = "^3.3.4"
+catkin-pkg = "^0.5.0"
```

Comment:
Do we actually still depend on catkin?

Reply:
colcon uses it, but colcon-common-extensions installs it, so indeed it should not be listed here